api/src/main/java/org/apache/iceberg/types/Types.java

```diff
@@ -543,6 +561,120 @@ public int hashCode() {
     }
   }
 
+  public static class GeometryType extends PrimitiveType {
+
+    private final String crs;
+
+    private GeometryType(String crs) {
+      this.crs = crs;
+    }
+
+    public static GeometryType get() {
+      return of("");
```

Comment:
why not , new GeometryType("")

Reply:
Fixed. Now we are using `null` as default value as requested. For now it is `new GeometryType(null)`.